<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>map</class>
 <widget class="QWidget" name="map">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1477</width>
    <height>709</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1477</width>
    <height>709</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1477</width>
    <height>709</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#map{
	
	background-color:rgb(48, 124, 72);
}
QLabel{
	color:white;
}

QPushButton{
	cursor:pointer;
}

QRadioButton{
	cursor:pointer;
}</string>
  </property>
  <widget class="QLabel" name="backmap">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-100</y>
     <width>1201</width>
     <height>811</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1201</width>
     <height>811</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1201</width>
     <height>811</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="back.qrc">:/newMap/mapImg/background.jpg</pixmap>
   </property>
  </widget>
  <widget class="QRadioButton" name="nanqu">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>600</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>南区操场</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>10</y>
     <width>251</width>
     <height>170</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="back.qrc">:/newMap/mapImg/sumz.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>190</y>
     <width>53</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Tips:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>230</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>1.左侧选择起点和终点计算最短路径</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>260</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>2.点击Clear重置路径</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>460</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>起点：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>540</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>终点：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calculatebtn">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>360</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #ffffff;
    border: 1px solid #dcdfe6;
    padding: 10px;
    border-radius: 5px;
}

QPushButton:hover {
    background-color: #ecf5ff;
    color: #409eff;
}

QPushButton:pressed, QPushButton:checked {
    border: 1px solid #3a8ee6;
    color: #409eff;
}

#calculatebtn {
    border-radius: 20px;
}</string>
   </property>
   <property name="text">
    <string>计算路径</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearbtn">
   <property name="geometry">
    <rect>
     <x>1330</x>
     <y>360</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #ffffff;
    border: 1px solid #dcdfe6;
    padding: 10px;
    border-radius: 5px;
}

QPushButton:hover {
    background-color: #ecf5ff;
    color: #409eff;
}

QPushButton:pressed, QPushButton:checked {
    border: 1px solid #3a8ee6;
    color: #409eff;
}

#clearbtn {
    border-radius: 20px;
}</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="qingzhen">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>540</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>清真食堂</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="sanshitang">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>470</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>三食堂</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="minzu">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>440</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#minzu{
	cursor:pointer;
}</string>
   </property>
   <property name="text">
    <string>民族学博物馆</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="nanhu">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>420</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>南湖操场</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="nanhuyuan">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>400</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>南湖园</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="shuangzita">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>310</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>双子塔</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="zhengmen">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>490</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>正门</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="zhongxinshitang">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>350</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>中心食堂</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="beiqu">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>220</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>北区公寓</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_start">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>462</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_over">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>543</y>
     <width>201</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_display">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>610</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="playmusic">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #ffffff;
    border: 1px solid #dcdfe6;
    padding: 10px;
    border-radius: 5px;
	cursor:pointer;
}

QPushButton:hover {
    background-color: #ecf5ff;
    color: #409eff;
}

QPushButton:pressed, QPushButton:checked {
    border: 1px solid #3a8ee6;
    color: #409eff;
}

#playmusic {
    border-radius: 30px;
}</string>
   </property>
   <property name="text">
    <string>play</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopmusic">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #ffffff;
    border: 1px solid #dcdfe6;
    padding: 10px;
    border-radius: 5px;
}

QPushButton:hover {
    background-color: #ecf5ff;
    color: #409eff;
}

QPushButton:pressed, QPushButton:checked {
    border: 1px solid #3a8ee6;
    color: #409eff;
}

#stopmusic {
    border-radius: 30px;
}</string>
   </property>
   <property name="text">
    <string>stop</string>
   </property>
  </widget>
  <widget class="QLabel" name="hero">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>530</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="back.qrc">:/newMap/mapImg/maonan.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="jieshao1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>350</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
    </font>
   </property>
   <property name="text">
    <string>介绍</string>
   </property>
  </widget>
  <widget class="QPushButton" name="jieshao2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>200</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
    </font>
   </property>
   <property name="text">
    <string>介绍</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>286</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>3.WASD控制人物上左下右边QEZC斜向</string>
   </property>
  </widget>
  <widget class="QLabel" name="caidan">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>70</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="back.qrc">:/newMap/mapImg/achang.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="showmap">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #ffffff;
    border: 1px solid #dcdfe6;
    padding: 10px;
    border-radius: 5px;
	cursor:pointer;
}

QPushButton:hover {
    background-color: #ecf5ff;
    color: #409eff;
}

QPushButton:pressed, QPushButton:checked {
    border: 1px solid #3a8ee6;
    color: #409eff;
}

#showmap {
    border-radius: 30px;
}</string>
   </property>
   <property name="text">
    <string>导航</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="back.qrc"/>
 </resources>
 <connections/>
</ui>
